src/Nethermind/Nethermind.Network/P2P/Subprotocols/Eth/Eth62ProtocolHandler.cs

```diff
@@ -299,6 +299,43 @@ private void Handle(NewBlockMessage newBlockMessage)
             }
         }
 
+        public override void NotifyOfNewBlock(Block block, SendBlockPriority priority)
+        {
+            if (priority == SendBlockPriority.High)
```

Comment:
switch priority and log error on default so we do not miss it ever in the future maybe

Reply:
still make default pass with Low prio behviour

Reply:
Something like this? https://github.com/NethermindEth/nethermind/pull/1490/commits/84635c582d859ab7c918a9dec8bf8fb80dfabc99#diff-ab73901f965652a9079f75ff5e03add0R304-R315